src/Symfony/Component/Validator/Tests/Constraints/IdenticalToValidatorTest.php

```diff
@@ -64,7 +64,7 @@ public function provideValidComparisons()
             array(null, 1),
         );
 
-        if (version_compare(PHP_VERSION, '>=', '5.5')) {
+        if (version_compare(PHP_VERSION, '5.5', '>=')) {
```

Comment:
We generally use ``PHP_VERSION_ID`` to check for the PHP version. It may be worth changing it here too